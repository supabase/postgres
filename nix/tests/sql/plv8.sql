create schema v;

-- create a function to perform some JavaScript operations
create function v.multiply_numbers(a integer, b integer)
  returns integer
  language plv8
as $$
  return a * b;
$$;

select
  v.multiply_numbers(3, 4);

drop schema v cascade;
