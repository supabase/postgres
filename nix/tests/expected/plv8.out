create schema v;
-- create a function to perform some JavaScript operations
create function v.multiply_numbers(a integer, b integer)
  returns integer
  language plv8
as $$
  return a * b;
$$;
select
  v.multiply_numbers(3, 4);
 multiply_numbers 
------------------
               12
(1 row)

drop schema v cascade;
NOTICE:  drop cascades to function v.multiply_numbers(integer,integer)
